src/app/modules/browse/browse-controller.js

```diff
@@ -83,23 +83,38 @@
                 params.bookOrBooks = _localize('book');
             }
 
+            var isFiltered = false;
+
             if (count === 0) {
-                message = _localize('There are no books that match your search for {shelf} {language} books', params);
+                $scope.resultsMessage = _localize('There are no books that match your search', params);
```

Comment:
No. The message is just reformatted into two lines. The first line gives a success message with number of results or a fail message. The second line lists out any filters.

For Example:
There are no books that match your search
 | language: English | keyword: "Taboo" |